cli/src/test/scala-2.12/coursier/cli/CliFetchIntegrationTest.scala

```diff
@@ -449,6 +516,20 @@ class CliFetchIntegrationTest extends FlatSpec with CliTestLib {
           // open jar and inspect contents
           val fileContents = Source.fromFile(urlInJsonFile).getLines.mkString
           assert(fileContents == "tada")
+
+//          testFile.delete()
+//
+//          assertThrows[Exception]({
```

Comment:
@dotordogh It depends on what you want to test… If you just want to test that the warning about `file:/` is printed, I guess tests for that particular point aren't needed…

Reply:
@alexarchambault I was hoping to test that the fetch would fail because the artifact would be missing after deleting it. Is there a way to do that with how we handle errors?

Reply:
I though something like would work, but the assert seems false…
```
val artifactOptions = ArtifactOptions(force = true)
val commonOpt = CommonOptions(jsonOutputFile = jsonFile.getPath)
val fetchOpt = FetchOptions(common = commonOpt, artifactOptions = artifactOptions)

// fetch with encoded url set to temp jar
val fetch = Fetch(
  fetchOpt,
  RemainingArgs(
    Seq(
      "e:f:g,url=" + encodedUrl
    ),
    Seq()
  )
)

assert(fetch.helper.anyError)
```
The idea would be just to try to avoid the `sys.exit` that makes the test exit.

Overall, the cli module should be reworked so that it doesn't call `sys.exit` too deep in the code this way, but that's how it is for now… :|